src/firebase/auth.js

```diff
@@ -0,0 +1,11 @@
+import firebase from "firebase/compat/app";
+import "firebase/compat/auth";
+import { signInAnonymously } from "firebase/auth";
+
+const auth = firebase.auth();
+
+export default auth;
+
+export function annonymousSignIn() {
+  return signInAnonymously(auth);
+}
```

Comment:
This is our auth import and setup file

I see that annonymousSignIn() is a custom function that returns the result of a function of from the auth library

I'm guessing it returns whether or not auth user has successfully signed into the database, enabling a connection between the database and the rest of the app

I think the spelling error is worth cleaning up here and elsewhere